recipes/napari-nanopyx/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "napari-nanopyx" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-nanopyx-{{ version }}.tar.gz
+  sha256: 2b2dd62703b8906c53bfc0b1c4b2a0215e6d9ba6b4558a1e831e6045edb29bab
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+
+test:
+  imports:
+    - napari_nanopyx
```

```suggestion
    - napari_nanopyx
    - napari_nanopyx.channel_registration
```

`__init__` is empty.